packages/teams-js/src/private/copilot/customTelemetry.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * @beta
+ * @hidden
+ * User information required by specific apps
+ * @internal
+ * Limited to Microsoft-internal use
+ * @module
+ */
+
+import { callFunctionInHost } from '../../internal/communication';
+import { ensureInitialized } from '../../internal/internalAPIs';
+import { ApiName, ApiVersionNumber, getApiVersionTag, getLogger } from '../../internal/telemetry';
+import { getCurrentTimestamp } from '../../internal/utils';
+import { runtime } from '../../public/runtime';
+
+const copilotTelemetryVersionNumber: ApiVersionNumber = ApiVersionNumber.V_2;
+const copilotLogger = getLogger('copilot');
+
+export enum Stage {
+  STAGE_E = 'E',
+}
+/**
+ * Sends custom telemetry data to the host.
+ *
+ * @param {Stage} [name=Stage.STAGE_E] - The stage of the telemetry data. Defaults to Stage.STAGE_E.
+ * @param {number} [timestamp=getCurrentTimestamp() ?? Date.now()] - The timestamp of the telemetry data. Defaults to the current timestamp.
+ * @returns {Promise<void>} - A promise that resolves when the telemetry data has been sent.
```

Comment:
This should probably say that the promise resolves when the host acknowledges that it has received the telemetry.

Reply:
fixed it.